tests/build/jvm-build.go

```diff
@@ -34,7 +34,7 @@ var (
 	testProjectRevision = utils.GetEnv("JVM_BUILD_SERVICE_TEST_REPO_REVISION", "34da5a8f51fba6a8b7ec75a727d3c72ebb5e1274")
 )
 
-var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jvm-build", "HACBS"), func() {
+var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jvm-build-service"), func() {
```

Comment:
Does [this](https://github.com/konflux-ci/e2e-tests/blob/a73e0e46707c69e26ee771a9d9700bb05b633148/magefiles/magefile.go#L421) need to be changed as well?